Tests/GravatarUITests/ProfileConfigurationTests.swift

```diff
@@ -106,9 +106,13 @@ final class TestProfileConfiguration: XCTestCase {
         var config = ProfileViewConfiguration.largeSummary(model: model)
         let view = config.makeContentView()
         view.translatesAutoresizingMaskIntoConstraints = false
-        config.palette = .light
+        config.palette = .custom {
+            Palette.light.withReplacing { avatarColors in
+                AvatarColors(border: .green, background: avatarColors.background)
```

Comment:
```suggestion
                avatarColors.withReplacing(border: .green)
```

Does the same thing but promotes the convenience method.

Reply:
nice!